index.bs

```diff
@@ -5343,27 +5343,36 @@ to obtain all of the above security benefits. However, one notable use case that
-securely, subsequent [=authentication=] [=ceremonies=] remain resistant to [=man-in-the-middle attacks=], i.e., benefit (3) is
-retained. Note, however, that such an attack would be easy to detect and very difficult to maintain, since any [=authentication=]
-[=ceremony=] that the same attacker does not or cannot tamper with would always fail.
-
-The [=[RP]=] SHOULD consider the above in its threat model when deciding its policy on what [[#sctn-attestation-types|attestation
-types]] to accept or whether to ignore [=attestation=].
+[INFORMATIVE]
 
-Note: The default [[#sctn-attestation-types|attestation type]] is [=None=], since the above issues will likely not be a major
-concern in most [=[RPS]=]' threat models. For example, the [=man-in-the-middle attack=] described above is more difficult than a
-[=man-in-the-middle attack=] against a [=[RP]=] that only uses conventional password authentication.
+When [[#registering-a-new-credential|registering a new credential]], the [=attestation statement=] might allow the [=[WRP]=] to,
+by unspecified procedures, derive guarantees about various qualities of the [=authenticator=], e.g., what kind of
+[=authenticator=] is being used, or how [=credential private keys=] are stored and protected. However, it is important to note
+that the [=attestation statement=] provides no way for the [=[RP]=] to verify that the [=attestation statement=] was generated by
+the [=authenticator=] the user intends, and not by a [=man-in-the-middle attack|man-in-the-middle attacker=], e.g., malicious code
+injected into [=[RP]=] script. The [=[RP]=] must therefore employ other means, e.g., TLS and related technologies, to protect the
+[=attestation statement=] from [=man-in-the-middle attacks=]. Failing to do so means that completing a [=registration ceremony=]
+constitutes a [=leap of faith=] with regard to the authenticity of the [=attestation statement=], including in particular the
+[=credential public key=], conveyed to the [=[RP]=], in which case benefit (3) above is no longer guaranteed.
+
+Under the assumption that a [=registration ceremony=] has been completed securely, and that the [=authenticator=] maintains
+confidentiality of the [=credential private key=], subsequent [=authentication ceremonies=] using that [=public key
+credential|credential=] are always resistant to [=man-in-the-middle attacks=], i.e., benefit (3) above remains guaranteed.
+
+The discussion above holds for all [=attestation types=]. In all cases it is possible for a [=man-in-the-middle
+attack|man-in-the-middle attacker=] to replace the {{PublicKeyCredential}} object, including the [=attestation statement=] and the
+[=credential public key=] to be registered, and subsequently tamper with future [=authentication assertions=] [=scoped=] for the
+same [=[RP]=] and passing through the same attacker. Note that such an attack would be detectable - since the [=[RP]=] has not
```

Comment:
Do you mean "would be detectable" or "would not be detectable"?

Reply:
I mean "would be detectable".